lib/utilities/lang.js

```diff
@@ -2,7 +2,9 @@
 // Thanks Yahoo!
 // http://yuilibrary.com/yui/docs/api/files/yui_js_yui-lang.js.html
 
-var TOSTRING     = Object.prototype.toString;
+/*eslint-disable */
```

Comment:
I figured that we shouldn't be worrying about other changing people's coding style (especially if we end up putting more YUI stuff in this fine down the road). Hence the global disable on eslint in just this file.